bedrock/firefox/templates/firefox/channel.html

```diff
@@ -24,8 +24,7 @@
 
 {% block body_id %}channel{% endblock %}
 
-{% block site_css %}{% endblock %}
-{% block extrahead %}
+{% block page_css %}
```

Comment:
Hmm, I wonder if we should leave this particular page as-is? Because it still runs the page JS, it makes accessing the download buttons a bit more tricky. It's still possible, but you have to know to click on the respective logo etc. Perhaps we need to class this one similar to the `/contact/` pages, as it is closely tied to the JS still?